res/css/views/settings/tabs/user/_SecurityUserSettingsTab.scss

```diff
@@ -27,14 +27,16 @@ limitations under the License.
 }
 
 .mx_SecurityUserSettingsTab {
-    .mx_SettingsTab_heading {
-        margin-bottom: 22px;
-    }
     .mx_SettingsTab_section {
         .mx_AccessibleButton_kind_link {
             padding: 0;
             font-size: inherit;
         }
+
+        .mx_SettingsTab_subsectionText--top {
+            color: $primary-content; // Same as mx_SettingsTab
+            margin-inline-end: 0; // align with devices panel
```

Comment:
All text in settings have right margin. I would say we keep this.

![image](https://user-images.githubusercontent.com/6216686/173295335-7cc71c39-bdc1-48c5-bb2a-824322b4b142.png)

Reply:
I think I copied the line from somewhere else not to break the current layout, let me check!

Reply:
This is the current layout, which does not have the right margin. Also the text color has been black.

![before](https://user-images.githubusercontent.com/3362943/173397352-2a7cdeec-b5ac-4bc2-9ace-1efed020da6d.png)

Should it be rather this?

| | |
|-|-|
|![after](https://user-images.githubusercontent.com/3362943/173398438-cd0c5510-fd8a-4afb-8c87-e240e89e16cb.png)|![after1](https://user-images.githubusercontent.com/3362943/173398715-88562bf1-76eb-4832-82b3-5e8788b7cfde.png)|